src/styles/add/stars.scss

```
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+
+  background-position: 50%;
+  background-repeat: no-repeat;
+  background-image: url(../images/star.svg);
+}
+
+.stars .stars__star:nth-child(-n + 4) {
```

```suggestion
.stars--1 .stars__star:nth-child(-n + 1),
.... ,
.stars--5 .stars__star:nth-child(-n + 5) {
```
Consider using styles from the previous task